js/picture-modal.js

```diff
@@ -31,31 +33,58 @@ const renderComments = (comments) => {
   commentsList.appendChild(fragment);
 };
 
+const showComments = () => {
+  const comments = commentsList.children;
+  const lowerCount = +socialCommentCount.dataset.commentCount;
```

Comment:
Я предпочитаю такой способ но это вкусовщина)
```suggestion
  const lowerCount = Number(socialCommentCount.dataset.commentCount);
```